doi-transformer/src/test/java/no/unit/nva/doi/transformer/CrossRefConverterTest.java

```diff
@@ -472,62 +529,191 @@ public void toPublicationPreservesOrcidWhenAuthorHasOrcid() throws InvalidIssnEx
     }
 
     @Test
-    @DisplayName("toPublication sets multiple affiliation labels when author has more affiliations name")
-    public void toPublicationSetsMultipleAffiliationLabelWhenAuthorHasMultipleAffiliationNameDuplicate()
-            throws InvalidIssnException {
-        setAuthorWithMultipleAffiliations(sampleInputDocument);
-        Publication actualDocument = toPublication(sampleInputDocument);
+    @DisplayName("toPublication sets PublicationContext to Book when crossref-type is book")
+    public void toPublicationSetsPublicationContextToBookWhjenCrossrefDocumentHasTypeBook()
+            throws InvalidIssnException, InvalidIsbnException {
+        CrossRefDocument sampleInputDocumentBook = createSampleDocumentBook();
+        Publication actualDocument = toPublication(sampleInputDocumentBook);
+        var publicationContext =  actualDocument.getEntityDescription().getReference().getPublicationContext();
 
-        List<Contributor> contributors = actualDocument.getEntityDescription().getContributors();
-        List<Organization> organisations =
-                contributors.stream()
-                .map(Contributor::getAffiliations)
-                .filter(Objects::nonNull)
-                .flatMap(Collection::stream)
+        assertTrue(publicationContext instanceof Book);
+    }
+
+    @Test
+    @DisplayName("toPublication sets ISBN when crossref-type is book")
+    public void toPublicationSetsIsbnWhenCrossrefDocumentIsTypeBook()
+            throws InvalidIssnException, InvalidIsbnException {
+        CrossRefDocument sampleInputDocumentBook = createSampleDocumentBook();
+
+        IsxnType type = IsxnType.PRINT;
+        Isxn printIsbnA = sampleIsxn(type, VALID_ISBN_A);
+        Isxn printIsbnB = sampleIsxn(type, VALID_ISBN_B);
+        List<Isxn> isbns = Arrays.asList(printIsbnA, printIsbnB);
+        sampleInputDocumentBook.setIsbnType(isbns);
```

Comment:
is the name setIsbnType correct here? should it be setTypedIsbn?  This could be fixed in another PR after the merging of this one.

Reply:
It's a construct of Crossref, named after the property, but can be renamed later...